config.py

```diff
@@ -1,25 +1,28 @@
 import os
-user = "root"
-password = "root"
-host = "localhost"
-database = "mapbot"
-port = "3306"
-key = "*Google_Cloud_API_key*"  # Will be provided by mentors
+import dotenv
 
-# your_path_to_stanford-corenlp-x.x.x.jar
-stanford_path_to_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2.jar"
+dotenv.load_dotenv()
 
-# your_path_to_stanford-corenlp-x.x.x-models.jar
-stanford_path_to_models_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2-models.jar"
+user = os.getenv('mapbotuser')
+password = os.getenv('mapbotpassword')
+host = os.getenv('mapbothost')
+database = os.getenv('mapbotdatabase')
+port = os.getenv('mapbotport')
+key = os.getenv('mapbotkey')
+
+stanford_path_to_jar = os.getenv('stanford_path_to_jar')
+
+stanford_path_to_models_jar = os.getenv('stanford_path_to_models_jar')
+
+javahome = os.getenv('javahome')
 
-# for eg. 'C:\\Program\ Files\\Java\\jdk1.8.0_201\\bin\\java.exe' or '/usr/local/openjdk-11/bin/java'
-javahome = '*your_path_to_jdk_bin_java.exe*'
 
 # DONOT CHANGE THE VALUES BELOW DURING INITIAL CONFIGURATION SET UP
 
-if os.getenv("DOCKER"):
+docker = os.getenv("DOCKER")
+if(docker=="Y"):
```

Comment:
A tiny nitpicking here:
Can do away with `()` braces in the `if` statement. Not exactly the Pythonic way of handling conditionals. :snake: 

> Note: Some of you may feel more comfortable using braces ({}) for code blocks and statements. The use or non-use of these braces in Python has been and continues to be a topic of vigorous debate. In spite of this, the core developers continue to reject any motion for braces based on Python's philosophy. In fact, if you open up Python and type from __future__ import braces you'll get a humorous error that can sum up Python's attitude towards braces...SyntaxError: not a chance.

Source: [here](https://en.wikiversity.org/wiki/Python_Concepts/If_Statement)